recipes/addfips/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "addfips" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e3525553f32a8c38c4e2130b473614b0e6e56846d8f3ed07162b14467481a957 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Try adding:

```yaml
  entry_points:
    - addfips=addfips.__main__:main
```

here to fix the Windows failure.

PS: consider using grayskull to create the recipe, it will add that automatically for you.

Reply:
Ahhhh, okay great. Thanks for that. I'll look into grayskull. It just sounds so scary. :skull: